src/components/Login/Login.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './Login.scss';
+import SignIn from './Sign-in/Sign-in';
+import SignUp from './Sign-up/Sign-up';
+
+const Login = () => {
+  return (
+    <div className='login'>
+      <SignIn />
+      <SignUp />
+    </div>
+  );
+};
+
+export default Login;
```

Comment:
component Login nie ma tutaj sensu nazwa wskazuje na component do logowania ktorym jest SignIn.

Zamiast tego powinien byc komponent Auth ktory wyswietla Albo SignIn albo SignUp w zaleznosci w jaki link kliknal uzytkownik a to pobieramy z routingu czyli url'a. Ta kwestie przegadamy jeszcze na koniec